app/dao/broadcast_service_dao.py

```diff
@@ -67,6 +68,13 @@ def set_broadcast_service_type(service, service_mode, broadcast_channel, provide
         status=INVITE_PENDING
     ).update({'permissions': VIEW_ACTIVITY})
 
+    # Revoke any API keys to avoid a regular API key being used to send alerts
+    ApiKey.query.filter_by(
+        service_id=service.id
```

Comment:
This will re-revoke any existing API keys as well, which is unnecessary / inaccurate.

Should we just revoke the active ones?

Reply:
Good point, fixed in 132411b